release_notes/Service_Fabric_ReleaseNotes_80CU1.md

```diff
@@ -0,0 +1,98 @@
+# Microsoft Azure Service Fabric 8.0 First Refresh Release Notes
+
+This release includes bug fixes and SDK improvements as described in this document.
+**This update will only be available through manual upgrades. Clusters set to automatic upgrades will not receive this update unless toggled to manual.**
+
+The following packages and versions are part of this release:
+
+| Service | Platform | Version |
+|---------|----------|---------|
+|Service Fabric Runtime| Ubuntu 16 <br> Ubuntu 18 <br> Windows | 8.0.515.1 <br>  8.0.515.1804 <br> 8.0.516.9590  |
+|Service Fabric for Windows Server|Service Fabric Standalone Installer Package | 8.0.516.9590 |
+|.NET SDK |Windows .NET SDK <br> Microsoft.ServiceFabric <br> Reliable Services and Reliable Actors <br> ASP.NET Core Service Fabric integration| 5.0.514  <br> 8.0.516 <br> 8.0.516 <br> 8.0.516 |
+|Java SDK  |Java for Linux SDK  | 1.0.6 |
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 11.0.0 |
+
+
+## Contents 
+
+Microsoft Azure Service Fabric 8.0 Release Notes
+
+* [Key Announcements](#key-announcements)
+* [Current And Upcoming Breaking Changes](#Current-And-Upcoming-Breaking-Changes)
+* [Service Fabric Runtime](#service-fabric-runtime)
+* [Service Fabric Common Bug Fixes](#service-fabric-common-bug-fixes)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+
+## Key Announcements
+* Service Fabric SDK will now support Startup Services as an alternative to Default Services specified in Application Manifest
```

Comment:
Do we have documentation on how to use Startup Services?

Reply:
As discussed, removed note about startup services.